.eslintrc.js

```diff
@@ -43,7 +40,6 @@ module.exports = {
   rules: {
     // Disallow Unused Variables
     // https://eslint.org/docs/rules/no-unused-vars
-    "no-unused-vars": ["error", { argsIgnorePattern: "^_" }],
     "@typescript-eslint/no-unused-vars": ["error", { argsIgnorePattern: "^_" }],
```

Comment:
The built-in `no-unused-vars` rule generates false positives. The `@typescript-eslint/no-unused-vars` rule is intended as a replacement that also works in JS files.